--- a/src/main/arithmetic.c	2015-03-26 16:15:17.000000000 -0700
+++ b/src/main/arithmetic.c	2015-08-10 00:00:34.000000000 -0700
@@ -214,7 +214,15 @@
 	/* There was a special case for y == 0.5 here, but
 	   gcc 4.3.0 -g -O2 mis-compiled it.  Showed up with
 	   100^0.5 as 3.162278, example(pbirthday) failed. */
+#ifdef _WIN32
+        if (y >= 0.0) {
+            return pow(x, y);
+        } else {
+            return 1 / pow(x, -y);
+        }
+#else
 	return pow(x, y);
+#endif
     }
     if (ISNAN(x) || ISNAN(y))
 	return(x + y);
